_test/general.test.php

```diff
@@ -58,4 +78,408 @@ public function test_plugin_conf() {
+
+        idx_addPage($pageId1, '', '');
+        idx_addPage($pageId2, '', '');
+
+        $conf = array();
+        $DokuwikiXMLExport = new DokuwikiXMLExport($conf);
+        $xml = new SimpleXMLElement($DokuwikiXMLExport->generateXMLExport(0, 20));
+
+        $start = implode('', ($xml->xpath('/findologic/items/@start')));
+        $count = implode('', ($xml->xpath('/findologic/items/@count')));
+        $total = implode('', ($xml->xpath('/findologic/items/@total')));
+
+        $expectedStart = 0;
+        $expectedCount = 2;
+        $expectedTotal = 2;
+
+        $this->assertEquals($expectedStart, $start, 'Expected start value should match "0" when DokuWiki has two pages.');
+        $this->assertEquals($expectedCount, $count, 'Expected count value should match "2" when DokuWiki has two pages.');
+        $this->assertEquals($expectedTotal, $total, 'Expected total value should match "2" when DokuWiki has two pages.');
+    }
+
+    /**
+     * Test to ensure that XML outputs basic export data.
+     */
+    public function test_xml_elements_equal_to_dokuwiki_page_data() {
+        $pageId = 'test123:test123:test123';
+        $pageTitle = 'test123';
+        saveWikiText($pageId, self::PAGE_CONTENT_PLACEHOLDER, '');
+        idx_addPage($pageId, '', '');
+
+        // Set title manually because you cant save it with the saveWikiText function.
+        $pageMetaTitle = array('title' => $pageTitle);
+        p_set_metadata($pageId, $pageMetaTitle);
+
+        $conf = array();
+        $DokuwikiXMLExport = new DokuwikiXMLExport($conf);
+        $xml = new SimpleXMLElement($DokuwikiXMLExport->generateXMLExport(0, 20));
+
+        $names = $xml->xpath('/findologic/items/item/names/name');
+        $name = $names[0];
+
+        $summaries = $xml->xpath('/findologic/items/item/summaries/summary');
+        $summary = $summaries[0];
+
+        $descriptions = $xml->xpath('/findologic/items/item/descriptions/description');
+        $description = $descriptions[0];
+
+        $ordernumbers = $xml->xpath('/findologic/items/item/allOrdernumbers/ordernumbers/ordernumber');
+        $ordernumber = $ordernumbers[0];
+
+        $urls = $xml->xpath('/findologic/items/item/urls/url');
+        $url = $urls[0];
+
+        $properties = $xml->xpath('/findologic/items/item/allProperties/properties/property');
+        $propertyKey = (string)$properties[0]->key[0];
+        $propertyValue = (string)$properties[0]->value[0];
+
+        $attributes = $xml->xpath('/findologic/items/item/allAttributes/attributes/attribute');
+        $attributeKey = (string)$attributes[0]->key[0];
+        $attributeValue = (string)$attributes[0]->values[0]->value[0];
+
+        $dateAddeds = $xml->xpath('/findologic/items/item/dateAddeds');
+        $dateAdded = (string)$dateAddeds[0]->dateAdded[0];
+
+        $expectedName = $pageTitle;
+        $expectedSummary = $expectedDescription = self::PAGE_CONTENT_PLACEHOLDER;
```

Comment:
The exporter should leave the description empty instead of duplicating it, otherwise the result scores will be skewed.

Reply:
@georgms I have created the issue #21